<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Текущий&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;разговор&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="conversationComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>424</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Поиск разговоров по городу</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="findConversationEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Поиск разговоров по номеру</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="findConversationEdit_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>531</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QLineEdit" name="cityEdit">
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="numEdit">
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="changeConversationButton">
      <property name="text">
       <string>Изменить тек. разговор</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="codeEdit">
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QTimeEdit" name="timeEdit">
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="deleteConversationButton">
      <property name="text">
       <string>Удалить тек. разговор</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Время разговора</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addConversationButton">
      <property name="text">
       <string>Добавить разговор</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Код</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="tariffEdit">
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Тариф</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Название города</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Дата разговора</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ном. тел. в этом городе</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_16">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Номер телефона</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="phoneNumEdit">
      <property name="inputMask">
       <string>+999 (99) 999-99-99</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="openFileButton">
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>741</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{

	color: #000;
}</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ConversationsButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>130</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Разговоры</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
